pkg/ast/ast.go

```diff
@@ -1313,6 +1341,33 @@ func (d *Document) RootOperationTypeDefinitionIsLastInSchemaDefinition(ref int,
 	}
 }
 
+func (d *Document) CreateRootOperationTypeDefinition(operationType OperationType, rootNodeIndex int) (ref int) {
```

Comment:
You're switching between Createxxx and Addxxx. Please decide on one naming pattern that fits with the rest of the library.